src/data/finset/basic.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2015 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Leonardo de Moura, Jeremy Avigad, Minchao Wu, Mario Carneiro
+Authors: Leonardo de Moura, Jeremy Avigad, Minchao Wu, Mario Carneiro, Hanting Zhang
```

Comment:
Note that the "Authors" list is not meant to include everyone who has contributed (git history does a good job of that), but rather is a list of the individuals who made significant contributions to the design and development of the code. See the explanation [here](https://leanprover-community.github.io/contribute/style.html#header-and-imports).

Reply:
Ah sorry. It's fixed now!

Reply:
No worries!